ppml/tdx/docker/client-image/spark-submit-with-ppml-tdx-k8s.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Check environment variables
+if [ -z "$SPARK_HOME" ]; then
+    echo "Please set SPARK_HOME environment variable"
+    exit 1
+fi
+
+if [ -z "$RUNTIME_K8S_SERVICE_ACCOUNT" ]; then
+    echo "Please set BIGDL_HOME environment variable"
+    exit 1
+fi
+
+if [ -z $RUNTIME_K8S_SPARK_IMAGE ]; then
+    echo "Please set BIGDL_HOME environment variable"
+    exit 1
+fi
+
+default_config="--conf spark.kubernetes.authenticate.driver.serviceAccountName=$RUNTIME_K8S_SERVICE_ACCOUNT \
+        --conf spark.kubernetes.container.image=$RUNTIME_K8S_SPARK_IMAGE \
+        --conf spark.kubernetes.executor.deleteOnTermination=false \
+        --conf spark.network.timeout=10000000 \
+        --conf spark.executor.heartbeatInterval=10000000 \
```

Comment:
these timeout is too large, do we need for tdx?

Reply:
Removed. We do not need on tdx.